src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -867,4 +867,7 @@ Additional information: {1}</value>
   <data name="Remove" xml:space="preserve">
     <value>Remove</value>
   </data>
+  <data name="Unfortunately_a_process_used_by_Visual_Studio_has_encountered_an_unrecoverable_error_We_recommend_saving_your_work_and_then_closing_and_restarting_Visual Studio" xml:space="preserve">
+    <value>Unfortunately, a process used by Visual Studio has encountered an unrecoverable error.  We recommend saving your work, and then closing and restarting Visual Studio.</value>
```

Comment:
We probably need to mention that features may now work properly until VS is restarted.